android-components/components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -109,6 +109,16 @@ object Glean {
         GleanCore.setExperimentInactive(experimentId = experimentId)
     }
 
+    /**
+     * Set configuration to override metrics' enabled/disabled state, typically from a remote_settings
+     * experiment or rollout.
+     *
+     * @param json Stringified JSON map of metrics and their associated `enabled` property.
+     */
+    fun setMetricsEnabledConfig(json: String) {
+        GleanCore.setMetricsEnabledConfig(json)
+    }
```

Comment:
Personally, I think I'd call the `toString()` in this method:

```suggestion
    fun setMetricsEnabledConfig(enabled: Map<String, Boolean>) {
        GleanCore.setMetricsEnabledConfig(JSONObject(enabled).toString())
    }
```